apps/system/js/app_window.js

```diff
@@ -1247,7 +1247,19 @@
    * once we need to display the image,
    * and revoke right away after we finish rendering the image.
    */
-  AppWindow.prototype._screenshotBlob = undefined;
+  AppWindow.prototype.__screenshotBlob = undefined;
+  Object.defineProperty(AppWindow.prototype, '_screenshotBlob', {
+    configurable: true,
+    get: function() {
+      return this.__screenshotBlob;
+    },
+    set: function(blob) {
+      this.__screenshotBlob = blob;
+      if (blob) {
+        this.publish('screenshotready');
+      }
+    }
+  });
```

Comment:
can we move this to the browser_mixin.js method?
(looks like the `renewCachedScreenshotBlob` method is never used)